src/fake-backend/fake-backend.js

```diff
@@ -0,0 +1,138 @@
+import people from "./people.json";
+import planets from "./planets.json";
+import films from "./films.json";
```

Comment:
There is a decent amount of mutating the files here in javascript. It would make more sense to change the files than constantly mutate them in js because we have control over the files.

I'll do that in another PR. I wish I had started with that assumption but I didn't 🤦 

For now this one MR fixes the API module to match exactly what it used to return without changing the json files.

Reply:
Originally I figured I'd leave the JSON files as is so we could migrate back to SWAPI once someone else decides to put a version of it live.